hack/latest-kind-images.sh

```diff
@@ -53,17 +53,18 @@ LATEST_123_TAG=$(latest_kind_tag "1\\.23")
 LATEST_124_TAG=$(latest_kind_tag "1\\.24")
 LATEST_125_TAG=$(latest_kind_tag "1\\.25")
 LATEST_126_TAG=$(latest_kind_tag "1\\.26")
-
+LATEST_127_TAG=$(latest_kind_tag "1\\.27")
 
 LATEST_122_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_122_TAG)
 LATEST_123_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_123_TAG)
 LATEST_124_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_124_TAG)
 LATEST_125_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_125_TAG)
 LATEST_126_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_126_TAG)
+LATEST_127_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_127_TAG)
 
-# k8s 1.27 is manually added to ensure that we use the exact documented tag as per kind recommendation
-LATEST_127_TAG=v1.27.3
-LATEST_127_DIGEST=sha256:3966ac761ae0136263ffdb6cfd4db23ef8a83cba8a463690e98317add2c9ba72
+# k8s 1.28 is manually added to ensure that we use the exact documented tag as per kind recommendation
+LATEST_128_TAG=v1.28.0
+LATEST_128_DIGEST=sha256:9f3ff58f19dcf1a0611d11e8ac989fdb30a28f40f236f59f0bea31fb956ccf5c
```

Comment:
This is the digest of the image rather than the multi-arch manifest.

```
$ crane digest --platform linux/amd64 docker.io/kindest/node:v1.28.0
sha256:9f3ff58f19dcf1a0611d11e8ac989fdb30a28f40f236f59f0bea31fb956ccf5c

$ crane digest docker.io/kindest/node:v1.28.0
sha256:b7a4cad12c197af3ba43202d3efe03246b3f0793f162afb40a33c923952d5b31
```

The previous digest **was** for the multi-arch digest:

```
$ crane digest docker.io/kindest/node:v1.27.3
sha256:3966ac761ae0136263ffdb6cfd4db23ef8a83cba8a463690e98317add2c9ba72
```

This didn't become apparent until I merged #6440 in which I saw this new digest and assumed that  kindest images were supposed to be referenced by image digest rather than multi-arch digest.

Reply:
@wallrj you are correct, these image digest might be wrong. I think I updated them using the "If it isn't failing, don't fix it" method.